Sources/Plasma/FeatureLib/pfPatcher/pfPatcher.cpp

```diff
@@ -569,12 +569,12 @@ void pfPatcherWorker::IHashFile(pfPatcherQueuedFile& file)
 
     // If you got here, they're different and we want it.
     PatcherLogYellow("\tEnqueuing '{}'", file.fServerPath);
-    plFileSystem::CreateDir(file.fClientPath.StripFileName());
+    plFileSystem::CreateDir(file.fClientPath.Normalize().StripFileName());
 
     // If someone registered for SelfPatch notifications, then we should probably
     // let them handle the gruntwork... Otherwise, go nuts!
     if (fSelfPatch) {
-        if (file.fClientPath == plFileSystem::GetCurrentAppPath().GetFileName()) {
+        if (file.fClientPath.Normalize() == plFileSystem::GetCurrentAppPath().GetFileName()) {
```

Comment:
This is probably the wrong way to do this. I wasn't entirely sure which functions needed the Windows path and which ones need the local path. I'm getting more comfortable with this code - but am intentionally leaving this messy so I can get feedback on which path should go where.

I considered just editing these functions to normalize paths but that could be risky or extra overhead where it isn't needed. But leaves open the possibility that other code paths may use Windows paths on a Mac or Linux host.

Reply:
I would prefer for te queued file to store a normalized copy of fClientPath. Needing to explicitly normalize something already called a "client path" is surprising and therefore bugprone.

Reply:
Done. Lemme know if it's sufficiently modern C++. I don't know any tricks besides brute forcing the plFileName constructor inside of the file constructor to get to the Normalize function.

Reply:
The "modern C++" way would probably be to change `fClientPath` to a `plFileName`, but that's outside the scope of what you're trying to accomplish, I think 😄